scripts/activation_responses.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python
+import argparse
+try:
+    import ConfigParser
+except ImportError:
+    import configparser as ConfigParser
+
+from os.path import isfile
+
+from pyne.mesh import Mesh, NativeMeshTag
+from pyne.dagmc import cell_materials, load, discretize_geom
+from pyne.r2s import resolve_mesh, irradiation_setup
+from pyne.alara import photon_source_to_hdf5, photon_source_hdf5_to_mesh,\
+    phtn_src_energy_bounds, response_to_hdf5, response_hdf5_to_mesh
+from pyne.mcnp import Meshtal
+
+config_filename = 'config.ini'
+alara_params_filename = 'alara_params.txt'
+
+config = \
+    """[general]
+# Specify whether this problem uses structured or unstructured mesh
+structured: True
+# List of requested response, seperated by commas.
+# Available options: 'decay_heat', 'specific_activity',
+# 'alpha_heat', 'beta_heat', 'gamma_heat', 'wdr', 'photon_source'.
+responses: decay_heat
+# wdr file for response wdr.
+wdr_file: IAEA.clearance
+
+[step1]
+# Path to MCNP MESHTAL file containing neutron fluxes or a DAG-MCNP5
+# unstructured mesh tally .h5m file.
+meshtal: meshtal
+# Tally number within the meshtal file containing the fluxes for activation.
+tally_num: 4
+# The name of the tag used to store flux data on the mesh. For unstructured
+# mesh this tag must already exist within the file specified in <meshtal>.
+flux_tag: n_flux
+# Path to the DAGMC material-laden geometry file (.h5m).
+geom: geom.h5m
+# If True the fluxes in the fluxin file will be printed in the reverse
+# order of how they appear within the flux vector tag. Since MCNP and
+# the Meshtal class order fluxes from low energy to high energy, this
+# option should be true if the transmutation data being used is
+# ordered from high-energy to low-energy.
+reverse: True
+# Number of rays to fire down each mesh row in each direction to calculate
+# cell volume fractions.
+num_rays: 10
+# If true, rays will be fired down mesh rows in evenly spaced intervals.
+# In this case <num_rays> must be a perfect square. If false, rays are fired
+# down mesh rows in random intervals.
+grid: False
+
+[step2]
+# List of decays times, seperated by commas. These strings much match exactly
+# with their counterparts in the phtn_src file produced in step1. No spaces
+# should appear in this line except the space between the time and the time unit
+# for each entry.
+decay_times:1E3 s,12 h,3.0 d
+"""
+
+alara_params =\
+    """material_lib alara_matlib
+element_lib nuclib
+data_library alaralib fendl3bin
```

Comment:
Does ALARA now work with fendl3?

Reply:
As far as I know, no.

Reply:
should probably change this default then:
```suggestion
data_library alaralib fendl2bin
```